pug/page-contents/buttons_content.html

```diff
@@ -25,7 +25,7 @@ <h3 class="header">Floating</h3>
         <br>
 
         <h5>Floating Action Button</h5>
-        <p>See the documentation on <a href="/floating-action-button.html">this page</a></p>
+        <p>See the documentation on <a href="/materialize/floating-action-button.html">this page</a></p>
```

Comment:
Thanks for the fix! 😃 

But I do think this should be `floating-action-button.html`, instead of `/materialize/floating-action-button.html`. Because when we run the documentation locally, it's accessed from `/docs` instead of `/materialize`.

Reply:
Sorry, I didn't try to build locally before proposing this fix. So the problem comes from the leading `/`. Should be even simpler to solve!